src/global/Constants.h

```diff
@@ -205,6 +218,10 @@ inline auto& RuntimeParameters() {
         SizeT<"cache-max-size-gb-single-entry">{5},
         SizeT<"lazy-index-scan-queue-size">{20},
         SizeT<"lazy-index-scan-num-threads">{10},
+        ensureStrictPositivity(
+            StreamableParameter<
+                ad_utility::ParseableDuration<std::chrono::seconds>,
+                "default-query-timeout">{30}),
```

Comment:
Is the type here in a way that you can write `30s` (with the correct namespace).